### ISSUE TITLE
%header.set(to,"[addressee]")% no longer working

### ISSUE THREAD

**Issue by JohnFLand** [RealRaven2000/SmartTemplate4#98]
A template of mine that worked fine before the 2020 updates to TB (I'm now on TB 78.5.0) no longer inserts an addressee. The text for the body of the email is inserted fine. Here's the entire template:

%header.set(to,"John Smith &lt;johnsmith@domain.com>")%
&lt;p>John --&lt;/p>
&lt;p>%cursor%&lt;/p>
%sig(html)%

Is there a workaround?

**Comment by RealRaven2000** [comment]
> A template of mine that worked fine before the 2020 updates to TB (I'm now on TB 78.5.0) no longer inserts an addressee. The text for the body of the email is inserted fine. Here's the entire template:
> 

```
%header.set(to,"John Smith <johnsmith@domain.com>")%
<p>John --</p>
<p>%cursor%</p>
%sig(html)%

```

Are you using that in a reply template or in a write (compose new)? I am not sure if I tested this a lot with "new mail" so that may be related.

That looks like  regression, needs to be fixed ASAP. I will post a beta here as soon as it's done!

**Comment by JohnFLand** [comment]
This template (and similar templates) is for composing new messages. 
Most of my new emails go to a small collection of individual addressees. 
Not having a problem with replies.

John Land

**Comment by RealRaven2000** [comment]
> This template (and similar templates) is for composing new messages. Most of my new emails go to a small collection of individual addressees. Not having a problem with replies. 

Thanks for clarifying. It would make sense as the addressing-widget has changed and I think there is some special trickery going on for that composition case. I see that there is some rewriting necessary in "modifyHeader( ) " function to deal with the new adressing widget. Will post a beta here soon.

**Comment by RealRaven2000** [comment]
OK, I got one for you - can you try out this version please:
[smartTemplate-fx-3.2pre2.zip](https://github.com/RealRaven2000/SmartTemplate4/files/5618815/smartTemplate-fx-3.2pre2.zip)

To install, download and drag the file into Thunderbird Add-ons Manager.

My test template (write new case)
```
%header.set(to,"John Smith <johnsmith@domain.com>")%
%header.set(cc,"Axel Smith <axesmith@domain.com>")%
<p>Dear %to(firstname)% --</p>
<p>%cursor%</p>
%sig(html)%
```

result:
![image](https://user-images.githubusercontent.com/6314674/100668335-af394400-3353-11eb-9334-3cc13574b58c.png)

I also test with an additional line
`%header.append(to,"Real Raven <rr2k@hotmail.com>")%`
and this will fill 2 addressees into the to line as expected:

![image](https://user-images.githubusercontent.com/6314674/100668587-0ccd9080-3354-11eb-9bf7-537b36d37067.png)

**Comment by JohnFLand** [comment]
Joy! Works fine, thanks for the prompt resolution!


John Land

**Comment by RealRaven2000** [comment]
Great, I will leave this open until I officially release 3.2 in case somebody else is looking for this!

**Comment by RealRaven2000** [comment]
Implemented in version [3.2](https://github.com/RealRaven2000/SmartTemplate4/releases/3.2) - released on 17/Dec/2020

This is currentlay awaiting official review on ATN.
